README.md

```diff
@@ -161,7 +161,7 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
       python3 -m venv --upgrade-deps venv
       source venv/bin/activate
       (venv) $ pip cache remove llama_cpp_python
-      (venv) $ pip install git+https://github.com/instructlab/instructlab.git@stable -C cmake.args="-DLLAMA_METAL=on"
+      (venv) $ pip install instructlab
```

Comment:
Do we need the `-C cmake.args="-DLLAMA_METAL=on"` flag or do we build natively with it?

Reply:
AFIK, no. It enabled by default.

Reply:
The cmake args are for llama_cpp_python which is a dependency of instructlab. So I think it is important to retain the cmake args here.

Reply:
Better cmake args are 

```
-C cmake.args="-DLLAMA_METAL=on;-DCMAKE_OSX_ARCHITECTURES=arm64;-DCMAKE_APPLE_SILICON_PROCESSOR=arm64"
```

which handles some odd macOS installs which were migrations from intel macs to arm macs which are some how configured to target x86_64. See https://github.com/instructlab/instructlab/discussions/1148

Reply:
The recent update of llama_cpp_python to 0.2.75 includes updates to llama.cpp which now defaults to "LLAMA_METAL=on". So we don't need this anymore.